build.cake

```diff
@@ -0,0 +1,108 @@
+#addin "Cake.FileHelpers"
+
+var target = Argument("target", "Default");
+var project = Argument("project", "*");
```

Comment:
This is part of the "run tests for a single project" feature. If this is not very important, we can skip this. The feature can be added as another PR?